analyticsdataserver/tests.py

```diff
@@ -46,7 +46,7 @@ def test_authentication_check_success(self):
     def test_health(self):
         self.assert_database_health('OK')
 
-    def assert_database_health(self, status):
+    def assert_database_health(self, status, status_code=200):
```

Comment:
FYI, you can use constants from Django REST framework's `status` module instead of bare status codes. It's not a big deal, but it's a little more readable.